src/daemon/daemon.h

```diff
@@ -123,9 +123,6 @@ public slots:
     grpc::Status version(grpc::ServerContext* context, const VersionRequest* request,
                          grpc::ServerWriter<VersionReply>* response) override;
 
-signals:
```

Comment:
Question: isn't there a corresponding connect call to remove too?

Reply:
Yes, and it has been removed in line 1569 in daemon.cpp above.

Reply:
Ah sorry I overlooked that